composer.json

```diff
@@ -18,10 +18,10 @@
         "php": ">=7.2",
         "ext-dom": "*",
         "guzzlehttp/guzzle": "^6.0.0",
-        "symfony/css-selector": "^3.0.0||^4.0.0",
-        "symfony/dom-crawler": "^3.0.0||^4.0.0",
-        "symfony/finder": "^3.0.0||^4.0.0",
-        "symfony/event-dispatcher": "^3.0.0||^4.0.0",
+        "symfony/css-selector": "^3.0.0||^4.0.0||^5.0.0",
+        "symfony/dom-crawler": "^3.0.0||^4.0.0||^5.0.0",
+        "symfony/finder": "^3.0.0||^4.0.0||^5.0.0",
+        "symfony/event-dispatcher": "^3.0.0||^4.0.0||^5.0.0",
```

Comment:
I am sorry, I missed this. That is why it would have been more foolproof if you had been able to merge my PR instead of re-implement it. But it seems we have everything now.

This line should read `"symfony/event-dispatcher": "^4.0.0||^5.0.0",`. Note the removal of version 3.

Reply:
Yes, you are absolutely right. Merging would have been cleaner. I should have avoided this. I hope we aren't missing anything due to this. I've dropped the v3 support for event-dispatcher.